website/src/views/conceptMap.js

```diff
@@ -23,7 +23,9 @@ import apiv2 from "../utils/apiv2";
  */
 export default function ConceptMap() {
     const [loading, setLoading] = useState(false);
-    const [studentMastery, setStudentMastery] = useState('000000');
+    const [CMhtml, setCMhtml] = useState('');
```

Comment:
```suggestion
    const [conceptMapHTML, setConceptMapHTML] = useState('');
```
non-class, non-static constants should start with a lower case letter

Reply:
Noted! Thanks for letting me know.